### ISSUE TITLE
Use local syntax highlighting and colorscheme

### ISSUE THREAD

**Issue by DyBev** [mistricky/codesnap.nvim#94]
Less of an issue, more of a feature request.

How much work would it be to use the local highlighting and colorscheme when taking a snapshot?

**Comment by mistricky** [comment]
Hi @DyBev, thx for this issue reporting, actually CodeSnap now using [syntect](https://github.com/trishume/syntect) to implement code highlight. Users may have their own color scheme, it's a lot of work if need to change the strategy of highlight code.

**Comment by DyBev** [comment]
I see thanks for pointing me towards the syntax highlighter, I can just add the syntax highlight file to that repo then it would solve the issue I'm facing where a snapshot can't be taken at all

**Comment by mistricky** [comment]
CodeSnap does not support custom code theme for now, but we have planned to support for custom code theme in the future.

**Comment by LZDQ** [comment]
Yeah why not just take a screenshot, fastest solution

**Comment by DyBev** [comment]
@LZDQ yes that is what I've been doing. I've moved away from the plugin now.

The comment isn't helpful. By your logic, why should this plugin exist in the first place, just take a screenshot after all you can just press cmd+shift+4 or cmd+shift+5 and take a screenshot or screen recording and you're done.

**Comment by augustocdias** [comment]
Maybe following the strategy of silicon.lua to build it at runtime:

https://github.com/0oAstro/silicon.lua/blob/main/lua/silicon/build_tmTheme.lua
